dattebayo/src/ErrorPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { useRouteError } from "react-router-dom";
+import { IError } from "./models/error";
+
+export default function ErrorPage() {
+	const error: IError = useRouteError() as IError;
```

Comment:
Love the error handling for your routes. Strongly typed, concise and clear to understand.